server/features/steps/steps.py

```diff
@@ -396,6 +396,7 @@ async def step_impl_fetch_from_provider_ingest(context, provider_name, guid):
 @when('we duplicate event "{event_id}"')
 def step_impl_when_we_duplicate_event(context, event_id):
     with context.app.test_request_context(context.app.config["URL_PREFIX"]):
+        # TODO-ASYNC[EventsService] - Convert this to async when function is updated to async
```

Comment:
You can easily convert behave steps to async.

```python
from behave.api.async_step import async_run_until_complete

@when('we duplicate event "{event_id}"')
@async_run_until_complete
async def step_impl_when_we_duplicate_event(context, event_id):
    ...
    # do async stuff here
```

Reply:
Missing `@async_run_until_complete` from implementation, otherwise behave will run this without awaiting it